src/main/java/core/basesyntax/service/ReportImpl.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service;
+
+import core.basesyntax.Main;
+import core.basesyntax.service.operation.OperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class ReportImpl implements Report {
+    private final Validator validator = new ValidatorImpl();
+    private final Storage handlersStorage = new Storage();
+
+    @Override
+    public String report(List<String> list) {
+        list.remove(0);
+        validator.validate(list);
+        StringBuilder report = new StringBuilder();
```

Comment:
declare this variable closer to the place, where you first use it

Reply:
did not understand which variable and where to transfer it

Reply:
StringBuilder report